addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -972,5 +972,77 @@ odoo.define('website.tour.form_editor', function (require) {
         },
     ]);
 
+    // Check that the editable form content is actually editable.
+    tour.register("website_form_editable_content", {
+        test: true,
+        url: "/",
+    }, [{
+            content: "Enter edit mode",
+            trigger: "a[data-action=edit]",
+        },
+        {
+            content: "Add a new 'Form' snippet",
+            trigger: "#oe_snippets .oe_snippet[name=Form] .oe_snippet_thumbnail",
+            run: "drag_and_drop #wrap",
+        },
+        {
+            content: "Select a form input field",
+            extra_trigger: ".s_website_form_field",
+            trigger: "section.s_website_form input",
+        },
+        {
+            content: "Check that the form field is not editable",
+            trigger: ".o_we_customize_panel:has(#o_we_editor_toolbar_container:contains(Inline Text):not(:visible))",
```

Comment:
This is too fragile: before your previous step, this trigger is true too as it checks that something is "not visible". This step is checked at every mutation which occurs after the previous step... maybe some are done before update the editor UI, I don't know. 

IMO, you can rather make an explicit check in a `run` function using the `isContentEditable` attribute (which checks the element and its ancestor if I am not mistaken).